flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamEdge.java

```diff
@@ -76,32 +76,68 @@
 
 	private final ShuffleMode shuffleMode;
 
-	public StreamEdge(StreamNode sourceVertex, StreamNode targetVertex, int typeNumber,
-			List<String> selectedNames, StreamPartitioner<?> outputPartitioner, OutputTag outputTag) {
-		this(sourceVertex,
-				targetVertex,
-				typeNumber,
-				selectedNames,
-				outputPartitioner,
-				outputTag,
-				ShuffleMode.UNDEFINED);
-	}
-
-	public StreamEdge(StreamNode sourceVertex, StreamNode targetVertex, int typeNumber,
-			List<String> selectedNames, StreamPartitioner<?> outputPartitioner, OutputTag outputTag,
-			ShuffleMode shuffleMode) {
+	private long bufferTimeout;
+
+	public StreamEdge(
+		StreamNode sourceVertex,
+		StreamNode targetVertex,
+		int typeNumber,
+		List<String> selectedNames,
+		StreamPartitioner<?> outputPartitioner,
+		OutputTag outputTag) {
```

Comment:
nit: indentation (other constructors too).

Reply:
I guess we suggested two options here in the past.

- Add one more indentation to distinguish with below codes, as you suggested above.

- Same indentation as this PR did, and add a separate empty line to distinguish with below codes. And we can see that many existing class constructors took this way, e.g. `Task`, `SingleInputGate`, `RemoteInputChannel`, etc.

So I will keep the current way :)